src/rust/plugin-registry/src/error.rs

```diff
@@ -31,4 +34,36 @@ pub enum PluginRegistryServiceError {
     NomadCliError(#[from] nomad::cli::NomadCliError),
     #[error("NomadJobAllocationError")]
     NomadJobAllocationError,
+    #[error("ArtifactTooLargeError")]
```

Comment:
this looks more like what i'd expect from a fmt::Debug than a fmt::Display, in that these are intended to be more human readable, but i understand if you don't want to break from the trend set by other variants here.